csharp/ql/src/Security Features/CWE-079/XSS.qhelp

```diff
@@ -11,17 +11,29 @@ without properly sanitizing the input first, allows for a cross-site scripting v
 </overview>
 <recommendation>
 
-<p>To guard against cross-site scripting, consider using contextual output encoding/escaping before
-writing user input to the page, or one of the other solutions that are mentioned in the
-references.</p>
+<p>
+To guard against cross-site scripting, consider using a library providing suitable encoding
+functionality, such as the <code>System.Net.WebUtility</code> class or the <code>AntiXSS</code> NuGet package,
```

Comment:
I didn't know this `AntiXSS` package. It doesn't seem to be maintained. Its last version was released on 6/2/2014, and its project website doesn't exist any longer: https://wpl.codeplex.com/. I don't think we should mention it.